packages/jellyfish-transaction-builder/src/provider.ts

```diff
@@ -1,6 +1,7 @@
 import BigNumber from 'bignumber.js'
 import { EllipticPair } from '@defichain/jellyfish-crypto'
 import { Vout } from '@defichain/jellyfish-transaction'
+import { ListUnspentQueryOptions } from '@defichain/jellyfish-api-core/dist/category/wallet'
```

Comment:
should not use `/dist`

Reply:
fixed https://github.com/BirthdayResearch/jellyfishsdk/pull/2156/commits/56612a2ff6bcd32af15ff96080b4caf8f678cdbd